packages/components/README.md

```diff
@@ -13,11 +13,12 @@ At the root of your React app, wrap your app in the `<Provider>` component:
 ```tsx
 // _app.tsx (or the root of your app)
 import { Provider } from '@web3-ui/components';
+import { NETWORK } from '@web3-ui/core';
 
 // Passing in a theme is optional
 function MyApp({ Component, pageProps }) {
   return (
-    <Provider theme={yourChakraTheme}>
+    <Provider theme={yourChakraTheme} network={NETWORKS.mainnet}>
```

Comment:
You don't/can't need to pass in a network to the components' provider. That's only needed in the core and hooks packages. :)

Reply:
@Dhaiwat10 yes I realised that after making this pr - surprised you merged it :D

The cause for confusion was that the network attribute has the wrong constraint. I made a little pr to update this https://github.com/Developer-DAO/web3-ui/pull/274 🙏